DigitalLearningSolutions.Web/Styles/trackingSystem/selectCourse.scss

```diff
@@ -0,0 +1,47 @@
+﻿.course-dropdown {

```

Comment:
These styles look like the rendered css output from core `.scss` files rather than `.scss`.
If you take a look at a file like `\\DigitalLearningSolutions.Web\Styles\shared\searchableElements\itemsPerPage.scss` - there are classes defined in there which inherit from common styles and the nhs style package.
Happy to talk this through with you if you like.

Reply:
Ah right sorry, I had just copied my css in here and didn't switch it to scss. 

I originally copied these styles from the Course Delegates page, where there is an identical Select Course box. Since the Course Delegates page uses these styles and lots of other ones, I've now moved the styles for the select box into selectCourse.scss and imported the selectCourse.scss file into the courseDelegates.scss file to reduce repeated code, but let me know if you think it should be organized differently.

Reply:
Thanks @livzorn that makes sense to me.